src/Todoist.Net.Tests/Services/TransactionTests.cs

```diff
@@ -44,5 +44,83 @@ public void CreateProjectAndCreateNote_Success()
 
             deleteTransaction.CommitAsync().Wait();
         }
+
+        [Fact]
```

Comment:
Could we please use `async/await` pattern here?
The old tests should be rewritten as well, but I don't have time for that now.

Reply:
Sure, I was planning also to rewrite them in my free time but I wanted to make sure that tests were not written synchronously on purpose.